src/torrent.cpp

```diff
@@ -121,9 +124,12 @@ namespace libtorrent
 		}
 		return ret;
 	}
-
 	constexpr int default_piece_priority = 4;
 
+	// holds seed rank priority duration in the ratio type which is used at
+	// seed rank calculation to prevent multible conversion and constructions
+	static constexpr seconds m_seed_rank_priority_duration = seconds(30 * 60);
```

Comment:
please remove ``static``. It's redundant since this definition is already in the anonymous namespace.
I use the ``m_`` prefix for members. This is not a member so it should not have the prefix.
Instead of calculating the number of second in 30 minutes, I think it would make more sense to be explicit:

``constexpr minutes seed_rank_priority_duration{30};``